### ISSUE TITLE
NVDA reads column headers marked up as TD in a THead in Adobe Reader/Acrobat

### ISSUE THREAD

**Issue by bramd** [nvaccess/nvda#9046]
In PDF THead has no semantic value and a TD inside a THead is still a regular data cell. However, NVDA+Adobe Reader/Acrobat seem to handle a TD in a THead as column header.

### Steps to reproduce:

1. Open attached PDF
1. Read the table either continuous or using cell by cell navigation

### Actual behavior:

NVDA reports the values in the first row as column headers.

### Expected behavior:

NVDA does not announce the values in the first row as column headers.

#### NVDA installed/portable/running from source:
Installed.

#### NVDA version:
NVDA alpha-16413,e3be1115

#### Windows version:
Windows 10 Version 1803 (OS Build 17134.407)

#### Name and version of other software in use when reproducing the issue:

* Adobe Acrobat Pro DC Version 2019.008.20081

[table-over-page-boundary.pdf](https://github.com/nvaccess/nvda/files/2667253/table-over-page-boundary.pdf)

#### Does the issue still occur after restarting your PC?
Yes.

#### Have you tried any other versions of NVDA?
No.
